test/system/250-systemd.bats

```diff
@@ -281,4 +281,30 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
     is "$output" "" "output should be empty"
 }
 
+# https://github.com/containers/podman/issues/13153
+@test "podman rootless-netns slirp4netns process should be in different cgroup" {
+    cname=$(random_string)
+    local netname=testnet-$(random_string 10)
+
+    # create network and container with network
+    run_podman network create $netname
+    run_podman create --name $cname --network $netname $IMAGE top
+
+    # run container in systemd unit
+    service_setup
+
+    # run second container with network
+    cname2=$(random_string)
+    run_podman run -d --name $cname2 --network $netname $IMAGE top
+
+    # stop systemd container
+    service_cleanup
+
+    # now check that the rootless netns slirp4netns process is still alive and working
+    run_podman exec $cname2 ping -c1 8.8.8.8
+
```

Comment:
If it is not running the ping will not work. I do not care much about the exact cgroup we just need to make sure it is not the unit cgroup because otherwise systemd will kill it when the unit is stopped.

Reply:
I added a extra check for the tap0 interface, this is only there as long as the process is alive